components/multiselect/multiselect.tsx

```diff
@@ -557,9 +557,9 @@ const paddedComponent = (WrappedComponent: any) => {
 const styles = {
     container: () => {
         return {
-            display: 'table-cell',
+            display: 'flex',
             verticalAlign: 'top',
-            width: '100%',
+            width: 'calc(100% - 76px)',
```

Comment:
curios on usage of 76px?

Reply:
76px makes up the width of cta that's next to the multiselect

Reply:
@michelengelen I moved away from `calc()` in favor if `flex: auto`

Reply:
@michelengelen I moved away from `calc()` in favor if `flex: auto`